content/soarian/dstu2/individuals/patient.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Patient | Soarian DSTU 2 API
+---
+ 
+# Patient
+
+* TOC
+{:toc}
+
+## Overview
+
+For the Patient resource query, the API returns a set of data stored for the patient as a result of Soarian admission, discharge, transfer (ADT) conversations.  The response represents the most current information about the patient charted in Soarian at the time of query.  
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_patient, :dstu2) %>
+
+## Custom Extensions
+The [bundle extension] can be returned, see possible codes in the [errors section] below.
+
+  Race Extension:
+
+ The URL for this custom extension is defined as  `http://hl7.org/fhir/StructureDefinition/us-core-race`
```

Comment:
This isn't a custom extension. Just list it under Extensions. EG: http://fhir.cerner.com/millennium/dstu2/individuals/patient/#extensions

Rename section to "Extensions"